compiler/src/dotty/tools/dotc/typer/ErrorReporting.scala

```diff
@@ -46,7 +46,13 @@ object ErrorReporting {
             errorMsg(msg, cx.outer)
         }
       } else msg
-    errorMsg(ex.show, ctx)
+
+      if (cycleSym.is(Implicit, butNot = Method) && cycleSym.owner.isTerm)
+        em"""cyclic reference involving implicit $cycleSym
```

Comment:
@felixmulder Line 2 and 3 of this message might be moved to an explanation. But we can also leave it like this, I think.